include/secp256k1.h

```diff
@@ -40,6 +40,26 @@ extern "C" {
 #  define SECP256K1_ARG_NONNULL(_x)
 # endif
 
+# if defined _MSC_VER || defined __CYGWIN__
+#  define SECP256K1_HELPER_DLL_IMPORT __declspec(dllimport)
```

Comment:
dllimport is only useful in case of global variables and C++ classes, not in case of functions. Removing dllimport support would simplify the patch.